### DISCUSSION TITLE
RSA-OAEP-256

### DISCUSSION THREAD

**Discussion by sanjays95** [panva/jose#29] [Q&A]
Any implementation/npm tool that supports RSA-OAEP-256 for the "alg" in JWE?

**Comment by panva** [comment]
node-jose supports it through a js based forge implementation fallback. 

@panva/jose support will come as soon as  [this](https://github.com/nodejs/node/pull/28335) lands in node core. I already have a branch working based on that node’s pr build

**Comment by panva** [comment]
This is now (v1.7.0) supported when node runtime is >=12.9.0
